### PR TITLE
1 increase account balance

### PR DESCRIPTION
**Pull request by Sushimaster2121**
i finished the thing

### CODE REVIEW

File: account-balances.txt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 Name    Balance
 Candace $8,441.82
-Francis $6,206.33
+Francis $68,545.89
```
Comment (TylerSeiford) [inline account-balances.txt:3]: This looks good

File: account-balances.txt [hunk 2]
```diff
@@ -1,6 +1,8 @@
 Name    Balance
 Candace $8,441.82
-Francis $6,206.33
+Francis $68,545.89
 Karl    $4,584.51
 Haile   $5,627.94
 Kim $7,913.01
+Jackson $100,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000,000
```
Comment (TylerSeiford) [inline account-balances.txt:7]: This isn't related to #1 ...

**Review by TylerSeiford — CHANGES_REQUESTED** [review]

File: account-balances.txt (hunk 2 above)
Comment (Noah-M-Reed) [reply account-balances.txt:7]: haha idiot

**Review by KieranMusser — APPROVED** [review]
Good enough

**Review by TylerSeiford — APPROVED** [review]
eh I guess it's good enough.  Enjoy being rich Jackson

**Pull request merged**